lib/create.js

```diff
@@ -200,10 +200,12 @@ module.exports.deleteInvalidTemplate = function(options) {
  *     - `e` {Error} is null unless there was an error.
  */
 
-module.exports.downloadTemplate = function(options, callback) {
-    var destinationPath = module.exports.templatePath(options),
-        url = 'https://github.com/phonegap/phonegap-app-hello-world/archive/master.tar.gz';
-
+module.exports.downloadTemplate = function(options, callback, url) {
+    var destinationPath = module.exports.templatePath(options);
+	if (url === undefined) {
+        var url = 'https://github.com/phonegap/phonegap-app-hello-world/archive/master.tar.gz';
+    }
+    
```

Comment:
Ah alright. I think we can remove this chunk of code entirely. I'm a big fan of discrete and focused work and the multiple templates is out of scope of patching the config.xml. Also, the PG Desktop App will likely be using the PG CLI for templates and the `connect-phonegap` create will be deprecated entirely. So we may never need to add template support to this function.